<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_Avatar_Page</class>
 <widget class="QWidget" name="Change_Avatar_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>527</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../src/resources/Resource.qrc">
    <normaloff>:/picture/transparent.png</normaloff>:/picture/transparent.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnUpload">
     <property name="text">
      <string>  上传照片</string>
     </property>
     <property name="icon">
      <iconset resource="../../../src/resources/Resource.qrc">
       <normaloff>:/svg/up_Avatar.svg</normaloff>:/svg/up_Avatar.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DragScrollArea" name="previewScrollArea">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>350</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="previewContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>348</width>
        <height>348</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPreview">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="CAP_groupBox_A">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnRotateCCW">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../src/resources/Resource.qrc">
          <normaloff>:/svg/left_rotation.svg</normaloff>:/svg/left_rotation.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderScale">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRotateCW">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../src/resources/Resource.qrc">
          <normaloff>:/svg/right_rotation.svg</normaloff>:/svg/right_rotation.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="CAP_Box">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">dragscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../src/resources/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
